src/components/MultiBarToggle.js

```diff
@@ -134,6 +137,14 @@ class MultiBarToggle extends Component {
                     >
                         {route.icon}
                     </AnimatedTouchable>
+                    {route.buttonlabel &&
```

Comment:
This text will be untouchable, are you sure that is correct?

Reply:
Yeah, the idea is to have the click only on the icon, once the text is too small to be clicked.